opendbc/dbc/generator/toyota/toyota_rav4_prime.dbc

```diff
@@ -0,0 +1,470 @@
+VERSION ""
+
+NS_ :
+    NS_DESC_
+    CM_
+    BA_DEF_
+    BA_
+    VAL_
+    CAT_DEF_
+    CAT_
+    FILTER
+    BA_DEF_DEF_
+    EV_DATA_
+    ENVVAR_DATA_
+    SGTYPE_
+    SGTYPE_VAL_
+    BA_DEF_SGTYPE_
+    BA_SGTYPE_
+    SIG_TYPE_REF_
+    VAL_TABLE_
+    SIG_GROUP_
+    SIG_VALTYPE_
+    SIGTYPE_VALTYPE_
+    BO_TX_BU_
+    BA_DEF_REL_
+    BA_REL_
+    BA_DEF_DEF_REL_
+    BU_SG_REL_
+    BU_EV_REL_
+    BU_BO_REL_
+    SG_MUL_VAL_
+
+BS_:
+
+BU_: XXX DSU HCU EPS IPAS CGW BGM
+
+BO_ 15 SECOC_SYNCHRONIZATION: 8 XXX
+ SG_ TRIP_CNT : 7|16@0+ (1,0) [0|65535] "" XXX
+ SG_ RESET_CNT : 23|20@0+ (1,0) [0|65535] "" XXX
+ SG_ AUTHENTICATOR : 35|28@0+ (1,0) [0|268435455] "" XXX
+
+BO_ 37 STEER_ANGLE_SENSOR: 8 XXX
+ SG_ STEER_ANGLE : 3|12@0- (1.5,0) [-500|500] "deg" XXX
+ SG_ STEER_RATE : 35|12@0- (1,0) [-2000|2000] "deg/s" XXX
+ SG_ STEER_FRACTION : 39|4@0- (0.1,0) [-0.7|0.7] "deg" XXX
+
+BO_ 170 WHEEL_SPEEDS: 8 XXX
```

Comment:
all this is new? or are they just different addrs?

Reply:
Yeah can we make a common DBC to reduce duplication? ADAS messages can be split out similar to what we did for Honda.

Reply:
It definitely needs a cleanup pass, just needed it out of critical path for this week's targets.

#1319